### ISSUE TITLE
About H5p contents on web page

### ISSUE THREAD

**Issue by ssyeh123** [tunapanda/h5p-standalone#39]
Does the standalone h5p contents can be moved to a web page for easy viewing? I had tried and without any success. Can someone give me some leads for further information. Thank you very much.

**Comment by ssyeh123** [comment]
I followed the instructions (1)unzip a h5p project (interactive-video) and move to workspace (2)create and change interactive-video.html as follows (3)copy entire h5p-standalone-master directory to a web server. After running and test the web page, I got a blank page without any information. Did I do something wrong? Thanks

....
<script type="text/javascript">
  const {
  H5P
  } = H5PStandalone;
  new H5P(document.getElementById('h5p-container'), 'workspace', {
  frameJs: '/dist/frame.bundle.js',
  frameCss: '/dist/styles/h5p.css',
  h5pContent: '/workspace/interactive-video'
  });
</script>
....
